### DISCUSSION TITLE
Why Is There No Exe?

### DISCUSSION THREAD

**Discussion by MaximosVeLOL** [KadeArchive/Kade-Engine#837] [Q&A]
i am working on a mod and this is hell without a exe

**Comment by Lucky-56** [comment]
what do you mean by that

**Comment by isakube** [comment]
`lime test <OS>`

if you're in VS Code and have the right extensions you can press Ctrl Shift B (Cmd for mac) to build to your target platform

**Reply by MaximosVeLOL** [reply]
im on windows

**Reply by MaximosVeLOL** [reply]
i used all commands with lime test windows and -debug and lime build windows

**Reply by ghost** [reply]
Have you gone into the source code folder, gone to export, then either "release" or "debug", then windows, then bin, then seen if theres one there?

**Reply by MaximosVeLOL** [reply]
i have and theres none

**Comment by noskincz** [comment]
copy the files to another file witch has an exe in it and delete the old one

**Reply by MaximosVeLOL** [reply]
but the new week doesn't show or work

**Reply by MaximosVeLOL** [reply]
i have a new idea just replace old week with new one im dumbass
